inference-engine/src/vpu/graph_transformer/src/middleend/passes/eliminate_redundant_conversions.cpp

```diff
@@ -29,6 +29,13 @@ void PassImpl::runForStage(const Model& model, const Stage& convert) {
     const auto input = convert->input(0);
     const auto output = convert->output(0);
 
+    // Remove converts added to unused inputs
+    if (output->usage() == DataUsage::Intermediate && output->numConsumers() == 0) {
```

Comment:
`... unused inputs` and `DataUsage::Intermediate` contradict each other. isn't that right?

Reply:
It's because the `Convert` stage was added to unused input, so now formally we have unused `output` of `Convert` stage.